dev/tasks/conda-recipes/clean.py

```diff
@@ -9,8 +9,23 @@
 
 
 VERSIONS_TO_KEEP = 5
-PLATFORMS = ["linux-64", "osx-64", "win-64"]
-PACKAGES = ["pyarrow", "arrow-cpp"]
+PACKAGES = [
+    "arrow-cpp",
+    "arrow-cpp-proc",
+    "parquet-cpp",
+    "pyarrow",
+    "pyarrow-tests",
+    "r-arrow",
+]
+PLATFORMS = [
+    "linux-64",
+    "linux-aarch64",
+    "osx-64",
+    "win-64",
+]
+EXCLUDED_PATTERNS = [
+    ["r-arrow", "linux-aarch64"],
```

Comment:
@kou Do you remember why this pattern is excluded?

Reply:
Because there are no packages for this combination.
It caused an error for this combination when I tried.